### PR TITLE
Dockerizing Managed Learn - SL Usermanagement Service

### PR DESCRIPTION
**Pull request by sivakarthikeyanoptit**

### CODE REVIEW

File: ansible/roles/stack-sunbird/templates/sl-usermanagement-service.env [hunk 1]
```diff
@@ -0,0 +1,52 @@
+HOST={{sl_usermanagement_host|default("")}}
```
Comment (rjshrjndrn) [inline ansible/roles/stack-sunbird/templates/sl-usermanagement-service.env:1]: are you sure the default value can be empty @sivakarthikeyanoptit ?

File: ansible/roles/stack-sunbird/templates/sl-usermanagement-service.env [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+APPLICATION_PORT={{sl_usermanagement_application_port|default('4101')}}
+LOG={{sl_usermanagement_log|default('debug')}}
```
Comment (rjshrjndrn) [inline ansible/roles/stack-sunbird/templates/sl-usermanagement-service.env:3]: Please add one line comment for all the variables
for example

```

# Connection string to the mysql db
# format: username@password:ip:port
secret_variable_connection_db: {{ db_connection_string | default ("") }}
```

**Review by rjshrjndrn — CHANGES_REQUESTED** [review]

File: ansible/roles/stack-sunbird/templates/sl-usermanagement-service.env (hunk 1 above)
Comment (sivakarthikeyanoptit) [reply ansible/roles/stack-sunbird/templates/sl-usermanagement-service.env:1]: i am not sure about it, if there is any change i have to do i will

File: ansible/roles/stack-sunbird/templates/sl-usermanagement-service.env (hunk 2 above)
Comment (sivakarthikeyanoptit) [reply ansible/roles/stack-sunbird/templates/sl-usermanagement-service.env:8]: yes rajesh changes had been done kindly review it

File: ansible/roles/stack-sunbird/templates/sl-usermanagement-service.env (hunk 1 above)
Comment (rjshrjndrn) [reply ansible/roles/stack-sunbird/templates/sl-usermanagement-service.env:2]: Please check with the dev and update it.

File: ansible/roles/stack-sunbird/templates/sl-usermanagement-service.env (hunk 2 above)
Comment (sivakarthikeyanoptit) [reply ansible/roles/stack-sunbird/templates/sl-usermanagement-service.env:8]: Hi rajesh, sl usermanagement devolper ask me remove, i am removing this one, and i am making pull request for sl sunbird

**Comment by sivakarthikeyanoptit** [comment]
sl usermanagement devolper ask me remove, i am removing this one, and i am making pull request for sl sunbird

**Pull request closed without merging**